api/src/main/java/org/openmrs/module/fhir/api/impl/RelatedPersonServiceImpl.java

```diff
@@ -33,6 +35,17 @@ public RelatedPerson getRelatedPerson(String uuid) {
 		return RelatedPersonStrageryUtil.getPersonStrategy().getRelatedPerson(uuid);
 	}
 
+	/**
+	 * Search related person by identifier
```

Comment:
We do not duplicate the javadoc. We simply use the `@see` annotation as you can see here https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java#L172